### ISSUE TITLE
namespace conflicts

### ISSUE THREAD

**Issue by jiangts** [noprompt/garden#82]
My setup has the following dependencies. 
                 [org.clojure/clojure "1.7.0"]
                 [org.clojure/clojurescript "0.0-3297"]
                 [reagent "0.5.0"]
                 [re-frame "0.4.1"]
                 [secretary "1.2.3"]
                 [cljs-ajax "0.3.14"]
                 [garden "1.2.5"]

When trying to run lein figwheel, I get

```
Compiling "resources/public/js/compiled/app.js" from ["src/cljs"]...
WARNING: Use of undeclared Var cljs.core/time at line 830 file:/.../.m2/repository/garden/garden/1.2.5/garden-1.2.5.jar!/garden/selectors.cljs
WARNING: time already refers to: cljs.core/time being replaced by: garden.selectors/time at line 830 file:/.../.m2/repository/garden/garden/1.2.5/garden-1.2.5.jar!/garden/selectors.cljs
WARNING: link at line 830 is being replaced at line 879 file:/.../.m2/repository/garden/garden/1.2.5/garden-1.2.5.jar!/garden/selectors.cljs
WARNING: Use of undeclared Var cljs.core/time at line 830 resources/public/js/compiled/out/garden/selectors.cljs
WARNING: time already refers to: cljs.core/time being replaced by: garden.selectors/time at line 830 resources/public/js/compiled/out/garden/selectors.cljs
WARNING: link at line 830 is being replaced at line 879 resources/public/js/compiled/out/garden/selectors.cljs
```

How can I fix this?
Thanks!

**Comment by jiangts** [comment]
I figured it out! Apparently I cannot use 
```clojure
(ns sb.styles.utils
  (:require [garden.core :refer [css style]]
                [clojure.string :as string]))
```
Instead, I just refer garden and use the functions with the namespace.

**Comment by jeluard** [comment]
It's probably worth looking at the root cause of this issue. Not sure if it should be expected or not. `time` is used [there](https://github.com/noprompt/garden/blob/master/src/cljx/garden/selectors.cljx#L813) but it looks fine? 

@noprompt what do you think?

**Comment by noprompt** [comment]
@jeluard It looks like this will be fixed by #85.

**Comment by jeluard** [comment]
This is now fixed in master.
